### PR TITLE
TBRANDS-30 : Quilted Image Block

### PR DESCRIPTION
**Pull request by ecarlisle**
## Description
Corrections were made to Quilted Image Block styling as Visual QA. 
Additional token changes were also made in Storybook, which can be testable from this PR.

Note: This PR corresponds to this PR in the `arc-themes-feature-pack` respository:
https://github.com/WPMedia/arc-themes-feature-pack/pull/222

### Visual QA
1. The Heading component had an incorrect left-margin for the default (mobile) viewport. It should now have a left-margin of 1rem (16px) and should vertically align with the content in the image overlays.
image
2. This block has a max width of 76rem (2016px) and is center-aligned when the width of the viewport increases above that value.

### Token Standardization Changes
1. Body font size was made 16px.
2. Spacing tokens were changed from `em` to use `rem`.
3. The comments showing pixel-equivalency are now moved to the line above token key-value pair. Initially they were on the same line, but prettier pushed them to the line below, which was confusing. Now the comments appear before the respective line, like this:
image

## Jira Ticket

- [TBRANDS-30](https://arcpublishing.atlassian.net/browse/TBRANDS-30)

## Acceptance Criteria

VQA feedback can be seen in the comments of [TBRANDS-30](https://arcpublishing.atlassian.net/browse/TBRANDS-30) to:
1. Have correct left-margin on mobile viewport as per this [Figma feedback](https://www.figma.com/file/giQllWgtzGk42KwK2XD9rY/Brand-Browsing-Page-Designs?node-id=1232%3A19606).
2. Block should have a max-width of 2016px and center alignment that enables margin on viewports larger than that width.

## Test Steps

1. Checkout this branch `git checkout {TBRANDS-30}`
2. View "Quilted Image Block" in Storybook by running `npm run storybook`.

### Before

1. Block heading doesn't have a 1rem `left-margin` on a mobile viewport.
2. Block doesn't have a 76rem `max-width` and doesn't center align in a larger container.

### After

1. Block heading has a 1rem `left-margin` on a mobile viewport.
2. Block has a 76rem `max-width` and center aligns in a larger container.

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [ ] Confirmed all the test steps a reviewer will follow above are working.
- [ ] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [ ] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [ ] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [ ] Confirmed this PR has unit test files
  - [ ] Ran `npm run test`, made sure all tests are passing
  - [ ] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [ ] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

**Comment by JackHowa** [comment]
good work on the screenshots and test instructions. I always like visual prs. to fix the issues deployed though, changes will need to be made to the feature pack. having the files in sync in storybook won't hurt, but the feature pack is where the changes need to be made 😄

**Comment by ecarlisle** [comment]
> good work on the screenshots and test instructions. I always like visual prs. to fix the issues deployed though, changes will need to be made to the feature pack. having the files in sync in storybook won't hurt, but the feature pack is where the changes need to be made 😄

Thank you very much, @JackHowa ! I totally agree and appreciate you pointing that out! I've created the feature pack PR and provided links to "the other PR" from each of the two.

File: .storybook/themes/commerce.scss [hunk 1]
```diff
@@ -168,13 +168,14 @@
 	$alias: (
 		"primary-color": var(--global-neutral-10),
 		"primary-font-family": "Inter",
+		"layout-content-max-width": 1216px,
```
Comment (nschubach) [inline .storybook/themes/commerce.scss]: Remove the units here.  This is going to break the responsive alias in category-carousel... see next comment as well.

File: .storybook/themes/commerce.scss [hunk 2]
```diff
@@ -288,6 +289,8 @@
 					),
 				),
 				quilted-image: (
+					margin: 0 auto,
+					max-width: var(--layout-content-max-width),
```
Comment (nschubach) [inline .storybook/themes/commerce.scss]:
```suggestion
					max-width: var(--layout-content-max-width + 0px),
```

Comment (nschubach) [reply .storybook/themes/commerce.scss]: This will allow you to use a unitless `--layout-content-max-width`

File: .storybook/themes/commerce.scss (hunk 1 above)
Comment (nschubach) [reply .storybook/themes/commerce.scss]:
```suggestion
		"layout-content-max-width": 1216,
```

File: .storybook/themes/commerce.scss (hunk 2 above)
Comment (ecarlisle) [reply .storybook/themes/commerce.scss]: @nschubach : Is that working for you? I've tried a unitless `layout-content-max-width` and used...
```
quilted-image: (
               margin: 0 auto,
               max-width: var(--layout-content-max-width + 0px),
```
... and it's still resulting in the 1440px width.
Also tried `calc(var(--layout-content-max-width) + 0px)` and having the same result.

Comment (ecarlisle) [reply .storybook/themes/commerce.scss]: @nschubach : Actually, multiplication worked for me.

Comment (nschubach) [reply .storybook/themes/commerce.scss]: Sorry, I was somewhat guessing and didn't realize I missed the `var()` or that `+` would not work in that case.  I was going ot say that you could use the `--layout-content-scale-width` but that would only apply for the desktop.  The mobile is full width from the looks of figma.  Though, I'm curious how this lines up with that given the different calculation.  We'll have to put them on a page together.

Comment (JackHowa) [reply .storybook/themes/commerce.scss]: multiplication's working for me as well. want me to push it up @ecarlisle ? https://github.com/WPMedia/arc-themes-blocks/pull/1321

File: .storybook/themes/commerce.scss [hunk 3]
```diff
@@ -104,58 +104,58 @@
 		"font-weight-8": 800,
 		"font-weight-9": 900,
 
-		"font-size-1": 0.625rem,
 		// 10px
-		"font-size-2": 0.75rem,
```
Comment (JackHowa) [inline .storybook/themes/commerce.scss]: can you revert some of these font-size changes? If this is keeping it in line with something else, we can maybe make it a different pr?

Comment (ecarlisle) [reply .storybook/themes/commerce.scss]: @JackHowa : I've got them updated. The only update remaining in that section is to have spacing in `rem` instead of `em`.

File: .storybook/themes/commerce.scss (hunk 1 above)
Comment (ecarlisle) [reply .storybook/themes/commerce.scss]: @nschubach : @JackHowa submitted a PR to this branch and we've got it fixed up. Thanks!

File: .storybook/themes/commerce.scss [hunk 4]
```diff
@@ -61,37 +61,38 @@
 		"neutral-9": #10151c,
 		"neutral-10": #000000,
 
-		"spacing-1": 0.25em,
```
Comment (JackHowa) [inline .storybook/themes/commerce.scss:64]: ok I see that makes more sense with the spacing change

**Review by JackHowa — APPROVED** [review]
Approve to look at storybook
